config/environments/development.rb

```diff
@@ -46,6 +46,12 @@
   # Send email in development mode?
   config.action_mailer.perform_deliveries = true
 
+  config.imap_settings = {
+    port: ENV['incoming_mail_port'],
+    address: ENV['incoming_mail_server'],
+    user_name: ENV['wherebot_user_name'],
+    password: ENV['wherebot_password']
```

Comment:
Environment variables are conventionally ALL_UPPERCASE. I think we should do that here (we don't on the action center, and I'm always conflicted about whether new env vars added there should match the convention of the site or the world at large :/)

Also, we use [dotenv](https://github.com/bkeepers/dotenv) to load environment variables on some of our other sites. It's great because it plays nicely with both vanilla rails and docker compose. If you add it, you can also add a .env.example file to document what environment variables are needed.

Reply:
Looks like we're currently using (Figaro)[https://github.com/laserlemon/figaro] to manage env variables in FAB.  Figaro is not currently maintained, and it would be great to move to dotenv, but I think I'm going to have to wait on that until after Wherebot is released 😅

Reply:
Ah! Works for me.